docusaurus/docs-internals/editor-controls/data-controls/editableSelect.md

```diff
@@ -0,0 +1,406 @@
+| `helper?.content`  | `string`                                                                                                                                                                                   |    -    | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `helper?.position` | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `choices`          | `Choices`                                                                                                                                                                                  |    -    | `Choices` is an array of `Choice` objects that are directly provided to the component.<br/><br/>**`Choice: { title: string; value: string; allowRemove?: boolean; allowEdit?: boolean; allowDuplicate?: boolean; }`**<br/><br/> Each `Choice` object defines the following properties:<br/>`allowRemove?` - Allows to remove the `Choice`.<br/>`allowDuplicate?` - Allows to duplicate the `Choice`.<br/>`allowEdit?` - Allows to edit the `Choice`.<br/>`title` - A string representing the display title of the choice.<br/>`value` - A string value representing the choice. This value is returned when the choice is selected. |
```

Comment:
for `choices`:

1. allowEdit?` - Allows to edit the `Choice` => review description because how you edit the whole Choice ? i think you mean that when `allowEdit` is `true` then user can change `Choice.title` or `Choice.value`
2. write required properties first:
- title
- value
- allowEdit
- allowDUplicate
- allowRemove